TicTacToe/BoardPosition.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicTacToe
+{
+    public class BoardPosition
+    {
+        private Dictionary<int, string> _positions  { get; set; } 
+
+        public BoardPosition()
+        {
+            _positions = new Dictionary<int, string>()
```

Comment:
This is an example of a data structure being the internal engine of an object. Because its the lowest level dependency in my solution, this single dictionary drives the entire game 3 objects down the rabbit hole. (see below comment for more on dependencies)